Core/src/main/resources/lang/messages_en.json

```diff
@@ -504,7 +504,7 @@
   "flag.flag_info_name": "<gray>Name: <gold><flag></gold></gray>",
   "flag.flag_info_category": "<gray>Category: </gray><gold><value></gold>",
   "flag.flag_info_description": "<gray>Description: </gray>",
-  "flag.flag_info_example": "<gray>Example: <click:suggest_command:<command> <flag> <value>><gold><command> <flag> <value></click></gold></gray>",
+  "flag.flag_info_example": "<gray>Example: <click:suggest_command:<command> <flag> <value>><gold><command> <flag> <value></gold></click></gray>",
```

Comment:
This change leads to 
![](https://i.imgur.com/wUkOxtl.png)
Unless this is a different caption I'm looking at.

Reply:
The change does fix exactly that behavior your screenshot is showing. you need to update that in the PlotSquared/lang/messages_en.json as well